app/locations/middleware.test.js

```diff
@@ -0,0 +1,119 @@
+const middleware = require('./middleware')
+
+describe('Locations middleware', function() {
+  let req, res, nextSpy
+  const userLocations = [
```

Comment:
so `userLocations` mock data is used in a couple of places. Could this be dry'd up?